CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Exposed new `dem_name` api parameter for RTC_GAMMA jobs
```

Comment:
maybe call out added `dynamodb.count_jobs`?  might be too low-level for the changelog